rfcs/0000-threat-intel.md

```diff
@@ -0,0 +1,119 @@
+# 0000: Cyber Threat Intelligence Fields
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **TBD** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+Elastic Security Solution will be adding the capability to ingest, process and utilize threat intelligence information for increasing detection coverage and helping analysts making quicker investigation decisions. Threat intelligence can be collected from a number of sources with a variety of structured and semi-structured data representations. This makes threat intelligence an ideal candidate for ECS mappings. Threat intelligence data will require ECS mappings to normalize it and make it usable in our security solution. This RFC is focused on identifying new field sets and values that need to be created for threat intelligence data. The goal is to utilize [STIX Cyber Observable data model](https://docs.oasis-open.org/cti/stix/v2.1/cs01/stix-v2.1-cs01.html#_mlbmudhl16lr) as a foundation for these new fields.
```

Comment:
Looking over the draft spreadsheet, it looks like `threat.*` may also make use of nesting of other ECS fields (`file.*`, `url.*`, `as.*`, etc.).

Not necessary to get into specifics of which fields may nest for stage 0, but one possible addition here would be mention that field reuse is also being explored along with adding new fields and categorization values.

Reply:
agreed. I will add language to reflect that we will explore field reuse as a priority before adding new fields.